app/helpers/application_helper/button/catalog_item_button.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::CatalogItemButton < ApplicationHelper::Button::Basic
+  include ApplicationHelper
```

Comment:
You dont have to `include ApplicationHelper` if you want to just call `role_allows?` you can just `:delegate :role_allows?, :to => :@view_context`